swag/currencies.py

```diff
@@ -86,3 +105,22 @@ def __neg__(self):
             return self
         else:
             raise InvalidStyleValue
+
+
+class Currency(str, Enum): 
+    """
+    Only used by slash command, 
+    Could be automaticly built thanks to Money children I guess ?
+    """
+
+    Swag = Swag(0).currency
```

Comment:
I think you can write Swag.currency, which means that this class member is useless
Also, please do not call a class constant using PascalCase, but using UPPER_SNAKE_CASE